fontforgeexe/fontview.c

```diff
@@ -656,16 +656,19 @@ int _FVMenuSaveAs(FontView *fv) {
     }
 #endif
 
-    ret = GWidgetSaveAsFileWithGadget8(_("Save as..."),temp,0,NULL,
+    utemp = utf82u_copy(temp);
+    ret = GWidgetSaveAsFileWithGadget((unichar_t*)_("Save as..."),utemp,0,NULL,
```

Comment:
Not your fault but I hate that `ret` is being used as a variable name that isn't the function's return value.